particles/Tutorial/Kotlin/3_JsonStore/JsonStore.arcs

```diff
@@ -0,0 +1,26 @@
+// Tutorial 3: JSON Store
+// Loads data stored in a JSON file.
+
+// Defines a new Entity called PersonDetails, with two fields.
+schema PersonDetails
+  Text name
+  Number age
+
+// Creates a data store of type PersonDetails, named PersonToGreetStore. The data will be loaded from the file data.json.
+store PersonToGreetStore of PersonDetails in 'data.json'
+
+particle JsonStoreParticle in '../build/bin/wasm32/mainDebugExecutable/main.wasm'
+  // This particle has an input parameter called inputData. We can use this parameter in the particle's JavaScript file.
+  in PersonDetails inputData
+  consume root
+
+recipe JsonStoreRecipe
+  // This line connects this recipe to the data store above. It also creates a local alias for it called "data", which is how we will refer to
```

Comment:
Let's keep the tutorial to a smaller column width for readability, since this is multi-line anyway.

Reply:
Rule of thumb: it should good on github :)